package.json

```diff
@@ -53,5 +59,8 @@
   },
   "peerDependencies": {
     "react": "^17.0.2"
+  },
+  "lint-staged": {
+    "*.{ts,js,css,md}": "prettier --write"
```

Comment:
I'm assuming that this has to do with the file extensions that prettier lints?
I think you'll need to add tsx, and jsx as well.

Reply:
Yes, this script is run by Husky. There's a pre-commit hook in the .husky directory that calls `npx lint-staged`. I went ahead and add those file extensions too. Nice catch!